coforma/swift-tech-challenge
# Issue/initial repo setup

# Description
Initial application scaffolding
Closes #1 

## Testing instructions
/documentation/product exists
 /documentation/design exists
 /documentation/research exists
 /documentation/engineering exists
 /documentation/engineering/adrs exists
 .github exists with issue templates
 License file exists
 Verify labels
Verify branch protections on main branch
